integration-test/erc721usermintable.contract.test-integration.js

```diff
@@ -0,0 +1,233 @@
+import { config as loadEnv } from 'dotenv';
+import ganache from 'ganache';
+import { BigNumber, utils } from 'ethers';
+import Auth from '../src/lib/Auth/Auth';
+import SDK from '../src/lib/SDK/sdk';
+import { TEMPLATES } from '../src/lib/NFT/constants';
+
+loadEnv();
+let sdk;
+let account;
+let server;
+let contractObject;
+let publicAddress;
+let owner;
+let thirdUser;
+let privateKeyPublicAddress;
+
+describe('E2E Test: User Payable NFT (write)', () => {
+  jest.setTimeout(120 * 1000);
+
+  beforeAll(async () => {
+    const options = {
+      wallet: {
+        accountKeysPath: 'integration-test/keys.json',
+      },
+      logging: {
+        quiet: true,
+      },
+    };
+
+    server = ganache.server(options);
+    await server.listen(8545);
+
+    // grab the first account
+    // eslint-disable-next-line global-require
+    const { addresses: addr, private_keys: pk } = require('./keys.json');
+    [owner, publicAddress, thirdUser] = Object.keys(addr);
+    const privateKey = pk[owner];
+    privateKeyPublicAddress = pk[publicAddress];
+
+    const rpcUrl = 'http://0.0.0.0:8545';
+    const chainId = 5;
+    const projectId = process.env.INFURA_PROJECT_ID;
+    const secretId = process.env.INFURA_PROJECT_SECRET;
+    const IPFS = { IPFSProjectID: '', IPFSProjectSecret: '' };
+
+    account = new Auth({
+      privateKey,
+      projectId,
+      secretId,
+      rpcUrl,
+      chainId,
+      IPFS,
```

Comment:
IPFS will be a separated module , with another auth, you can delete this line .
example: 
`const ipfs = new IPFS( ...... ) , ipfs,uploadFIle(...)`